xml/signals/Reading-1954/appearance-SL-2-high-abs.xml

```diff
@@ -38,60 +38,43 @@
 
   <appearances>
 
-    <appearance>
-      <aspectname>Clear alt</aspectname>
-      <show>green</show>
-      <show>green</show>
-      <reference>Page 92</reference>
-      <comment>Unofficial Rule and Aspect Name to allow for optional appearance</comment>
-      <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-281-alt.gif</imagelink>
-    </appearance>
```

Comment:
Could it cause problems for users if this is removed?

Reply:
Yes, if a user happens to make use of the alternative aspect.

Note that some signaling systems include the "alternate" aspect definitions, but do not provide proper "mapping" such that SML can actually make use of them.  That makes the "alternate" aspects difficult to make use of, unless one writes Logix/Logix-NG/scripting to "swap-in" the alternate aspect in place of the normal one.  

FWIW, my recent AAR-1946 updates make "Clear Alt" a more-usable mast aspect, along with other "improvements" to that signaling system.